File: nest.core.view.main/wwwroot/src/Aplicacion/Servicios/ModuloService.js
function ModuloService() {
    this.path = "/security/Modulo"
    const client = new httpClientApi();

    this.ObtenerTodos = async () => {
        const respuesta = await client.executeGet(this.path);
        return respuesta;
    };

    this.ObtenerPorId = async (id) => {
        const respuesta = await client.executeGet(`${this.path}/${id}`);
        return respuesta;
    };

    this.ObtenerPorUnaPropiedad = async (filter) => {
        const respuesta = await client.executePost(`${this.path}/filter`, filter);
        return respuesta;
    };

    this.Agregar = async (registro) => {
        const respuesta = await client.executePost(`${this.path}`, registro);
        return respuesta;
    };

    this.Modificar = async (id, registro) => {
        const respuesta = await client.executePut(`${this.path}/${id}`, registro);
        return respuesta;
    };

    this.Eliminar = async (id, registro) => {
        const respuesta = await client.executeDelete(`${this.path}/${id}`);
        return respuesta;
    };
}

File: nest.core.aplicacion.security/Aplicacion/ModuloService.cs
using nest.core.dominio.Aplicacion.Modulo;
using nest.core.dominio.Aplicacion.Modulo.Repository;

namespace nest.core.aplicacion.security.Aplicacion
{
    public class ModuloService
    {
        private readonly IModuloRepository repository;
        public ModuloService(IModuloRepository repository) 
        { 
            this.repository = repository;
        }

        public Task<Modulo> ObtenerPorId(int id) => this.repository.ObtenerPorId(id);
        public Task<List<Modulo>> ObtenerTodos() => this.repository.ObtenerTodos();
        public Task<List<Modulo>> ObtenerPorUnaPropiedad(Dictionary<string, object> filtros) => this.repository.ObtenerPorUnaPropiedad(filtros);
        public Task<Modulo> Agregar(ModuloCrearDto entry) => this.repository.Agregar(entry);
        public Task<Modulo> Modificar(int id, ModuloCrearDto entry) => this.repository.Modificar(id, entry);
        public Task Eliminar(int id) => this.Eliminar(id);
    }
}
